src/cpp/fastdds/xtypes/type_representation/TypeObjectUtils.cpp

```diff
@@ -2050,6 +2050,51 @@ void TypeObjectUtils::plain_collection_type_identifier_header_consistency(
     }
 }
 
+void TypeObjectUtils::plain_map_type_identifier_header_consistency(
+        const PlainCollectionHeader& header,
+        const TypeIdentifier& element_identifier,
+        const TypeIdentifier& key_identifier)
+{
+    switch (header.equiv_kind()){
```

Comment:
Not really sure why the linters are not complaining

```suggestion
    switch (header.equiv_kind())
    {
```

Reply:
fixed in [ba259bd](https://github.com/eProsima/Fast-DDS/pull/4496/commits/ba259bd6ba580ab197d666dc0867ba51c86bfebf)